### ISSUE TITLE
.Net Core 2 Support

### ISSUE THREAD

**Issue by wesleyteixeira** [DbUp/DbUp#269]

**Comment by AdrianJSClark** [comment]
Unfortunately at the moment .Net Core support is in a bit of limbo due to time available from maintainers. If you want to check out where it is at, grab the [4.0 Release Branch](https://github.com/DbUp/DbUp/tree/release/4.0.0). There's some discussion of the .Net Core support specifically in #174 and if you have comments on the wider 4.0 release itself add some comments to #176.

**Comment by kspearrin** [comment]
Hey guys,
Some good news! I just tried DbUp in a .NET Core 2.0 project and it works just fine. This is due to the library shimming that .NET Core 2.0 does to make old libraries compatible.

In addition to adding `DbUp` library to your project, you just need to add `System.Data.SqlClient` (4.4.0) from nuget as well.

I haven't tested everything, but here's the code/features I am using successfully:

```
var upgrader = DeployChanges.To
    .SqlDatabase(MY_CONNECTION_STRING)
    .JournalToSqlTable("dbo", "Migrations")
    .WithScriptsAndCodeEmbeddedInAssembly(
        Assembly.GetExecutingAssembly(),
        (string s) => s.Contains($".Scripts."))
    .WithTransactionPerScript()
    .LogToConsole()
    .WithExecutionTimeout(new TimeSpan(0, 0, 90))
    .Build();

var result = upgrader.PerformUpgrade();
```

👍

**Comment by wesleyteixeira** [comment]
Thanks! Worked like a charm!

**Comment by droyad** [comment]
We have published `4.0.0-beta0002` which supports `netstandard1.3` (and `netcoreapp1.0`). The `Dbup` package is obsolete, use `dbup-sqlserver` (MS SQL Server) or the package relevant to your database.

**Comment by nulltoken** [comment]
> We have published 4.0.0-beta0002 which supports netstandard1.3 (and netcoreapp1.0). The Dbup package is obsolete, use dbup-sqlserver (MS SQL Server) or the package relevant to your database.

@droyad ❤ Gave it a try. It works flawlessly in my context. Thanks a lot!
